### PR TITLE
Added Gold Infinity

### PR DESCRIPTION
**Pull request by meganyyu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ Credits for images:
 * Cursed Pendant Of Luck: made by BenjaminAmos (own work)
 * Gold Badge of Luck: made by Anish Sharma (own work)
 * Wooden Ring of Strength: made by PokeyOne (own work)
+* Gold Infinity: https://cdn.pixabay.com/photo/2016/11/19/00/17/infinity-1837437_1280.png
```
Comment (oniatus) [inline README.md]: Direct linking to pixabay won't work. I think it is this one?
https://pixabay.com/en/infinity-infinite-repeating-loop-1837437/

**Review by oniatus — CHANGES_REQUESTED** [review]
The item works (tested), only a small change to the image link requred.

File: README.md (hunk 1 above)
Comment (meganyyu) [reply README.md]: Yup, that's right.

**Comment by meganyyu** [comment]
Updated README.md

**Pull request merged**